api/internal/handler/data_loader/route_export_test.go

```diff
@@ -1903,10 +1904,66 @@ func TestExportRoutesSameURI(t *testing.T) {
 	assert.True(t, getCalled)
 }
 
+func TestExportRoutesParameterEmpty(t *testing.T) {
+	// Error test when pass parameter is null
+	r1 := `{
+		"uris": ["/test-test"],
+		"name": "route_all",
+		"desc": "所有",
+		"methods": ["GET"],
+		"hosts": ["test.com"],
+		"status": 1,
+		"upstream": {
+			"nodes": {
+				"172.16.238.20:1980": 1
+			},
+			"type": "roundrobin"
+		}
+}`
+	var route *entity.Route
+	err := json.Unmarshal([]byte(r1), &route)
+	assert.Nil(t, err)
+	mStore := &store.MockInterface{}
+	mStore.On("Get", mock.Anything).Run(func(args mock.Arguments) {
+	}).Return(route, nil)
```

Comment:
IMO, mStore.Get will never be called if the input.IDs is "".
So we can remove the mock here.

Reply:
done. Thks for review